packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/testProcessLineCreatesParagraph.st

```diff
@@ -1,7 +0,0 @@
-tests
-testProcessLineCreatesParagraph
```

Comment:
We removed 3 tests concerning the old `processLine:` because they effectively test the block structure on a more granular level. We also found it strange that they only use headings and paragraphs.